packages/kit/src/runtime/server/page/fetch.js

```diff
@@ -176,6 +175,25 @@ export function create_fetch({ event, options, state, route, prerender_default }
 
 			const external_request = new Request(requested, /** @type {RequestInit} */ (opts));
 			response = await options.hooks.externalFetch.call(null, external_request);
+
+			if (opts.mode === 'no-cors') {
```

Comment:
Note for later: If we add `fetch` to `load` of `+page/layout.server` at some point, this needs to be checked because on the server it's ok to fetch cross origin.

Reply:
I was wondering if it should work, to do a fetch in +layout.server.js or +page.server.js, because they are both server side and should not care about CORS.